### PR TITLE
Fix scopes renderer when resource has only optional scopes and their conditions are false

### LINKED ISSUE #5578: conditional scopes output strange output in index page
After we upgraded to 1.4.0 we see that conditional scopes are outputting some strange debug info in the place where scopes should be. 
```
scope 'test_scope', if: -> { false } do |scope|
    scope.where(...)
  end 
```

This outputs:
screenshot 2018-11-20 11 02 00

### PR DESCRIPTION
**Pull request by Looooong**
This PR resolves #5578 .

### CODE REVIEW

**Comment by Looooong** [comment]
@deivid-rodriguez Maybe I'm over-testing this. One scenario should be enough to cover the issue.

**Comment by Looooong** [comment]
I have updated the scenario.

File: features/step_definitions/index_scope_steps.rb [hunk 1]
```diff
@@ -36,7 +36,11 @@
   expect(page).to     have_css ".scopes .scope-group-#{group} .#{name2}"
 end
 
-Then"I should see an empty group with the scope {string}" do |name|
+Then "I should see an empty group with the scope {string}" do |name|
```
Comment (deivid-rodriguez) [inline features/step_definitions/index_scope_steps.rb]: Sorry, I forgot to mention this. I landed this separately, so if you could rebase on top of latest master to remove this bit from the final diff, that'd be great :+1:.

Comment (Looooong) [reply features/step_definitions/index_scope_steps.rb]: I will do it.

**Pull request merged**